### PR TITLE
Atomic Hosting: Display data loss warning

### LINKED ISSUE #37257: Hosting: Add Support Button or Data Loss Warning
Updating files using SFTP or modifying the DB may result in potential data loss. As suggested by @automattic-ian let's warn or provide support help before folks try to modify their site.

Some suggestions by @shaunandrews 
screen-shot-2019-11-01-at-10 54 10-am

screen-shot-2019-11-01-at-11 01 20-am

cc @olaolusoga @apeatling

### PR DESCRIPTION
**Pull request by mmtr**
#### Changes proposed in this Pull Request

Displays a data loss warning in the Atomic Hosting Panel providing support help before folks try to modify their site

| Desktop | Tablet | Mobile |
|---|---|---|
| Screen Shot 2019-11-11 at 17 20 23 | Screen Shot 2019-11-11 at 17 20 10 | Screen Shot 2019-11-11 at 17 19 56 |

#### Testing instructions

* Switch to an Atomic site created after Oct 31, 2019.
* Go to "Manage > SFTP & MySQL".
* Note the "Unsure of what all this means?" warning.
* Verify the button redirects to the contact page (`/help/contact`).

Fixes #37257

### CODE REVIEW

**Comment by mmtr** [comment]
The original mockup in #37257 displays a list of avatars to the right of the contact button:

screen-shot-2019-11-01-at-11 01 20-am


However, I don't think that's doable right now (I'm not aware of any service or tool that can provide HEs avatars). 

I decided to don't include any image, but we could display an alternative illustration like [this](https://wordpress.com/calypso/images/customer-home/happiness.png) if desired . cc @olaolusoga  @shaunandrews

File: client/my-sites/hosting/style.scss [hunk 1]
```diff
@@ -12,17 +12,25 @@
 	.card {
 		display: flex;
 		flex-direction: row;
-		width: calc( 100% - 10px );
+		width: 100%;
```
Comment (mmtr) [inline client/my-sites/hosting/style.scss]: All these changes have been needed to ensure the data loss warning is aligned with the card above.

**Comment by glendaviesnz** [comment]
When I test this the mobile view is different from your screenshot - there is no padding/margin on the cards or the warning:

Screen Shot 2019-11-11 at 12 09 09 PM

**Comment by mmtr** [comment]
Nice catch @glendaviesnz! My screenshot was actually from a table/mid-size view 🙂 Fixed in 4ff4d7e 
 and summary updated with more accurate screenshots.

**Comment by shaunandrews** [comment]
> However, I don't think that's doable right now (I'm not aware of any service or tool that can provide HEs avatars). I decided to don't include any image, but we could display an alternative illustration like this if desired.

The list of avatars already exists on our current help screen: https://wordpress.com/help

**Comment by mmtr** [comment]
> The list of avatars already exists on our current help screen: https://wordpress.com/help

TIL, thanks! Seems pretty easy to implement, will update this PR in a bit.

**Comment by shaunandrews** [comment]
Can we provide a max-width for this? Right now it seems to take up 100% of the available width which leads to a line-length that is hard to read. Perhaps we reduce it to more like 650px or so, keeping it left aligned.

**Comment by mmtr** [comment]
> Can we provide a max-width for this? Right now it seems to take up 100% of the available width which leads to a line-length that is hard to read. Perhaps we reduce it to more like 650px or so, keeping it left aligned.

Sure! I finally used 700px so the last avatar is better aligned with the text (at least in English):
Screen Shot 2019-11-12 at 11 42 19

**Pull request merged**